pom.xml

```diff
@@ -670,8 +676,10 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <hyracks.fullstack.version>0.3.1</hyracks.fullstack.version>
```

Comment:
Yes of course @prestoncarman. This will be removed when VXQuery moves to next Hyracks version (0.3.1 - because this is the only version having [hyracks-http](https://mvnrepository.com/artifact/org.apache.hyracks/hyracks-http))